examples/v2/chat2.nim

```diff
@@ -162,10 +165,22 @@ proc processInput(rfd: AsyncFD, rng: ref BrHmacDrbgContext) {.async.} =
   let listenStr = $peerInfo.addrs[0] & "/p2p/" & $peerInfo.peerId
   echo &"Listening on\n {listenStr}"
 
+  let topic = cast[Topic](DefaultContentTopic)
+  let multiAddr = MultiAddress.initAddress(conf.storenode)
+  let parts = conf.storenode.split("/")
+
+  node.wakuStore.setPeer(PeerInfo.init(parts[^1], [multiAddr]))
```

Comment:
Maybe this is where the `Error: unhandled exception: index out of bounds, the container is empty [IndexError]` comes from? If storenode isn't specified correctly